### DISCUSSION TITLE
Facing issue with Angular 16

### DISCUSSION THREAD

**Discussion by rahulkm561** [WalletConnect/walletconnect-monorepo#4096] [Developer Support] [answered]
Hello Team, 

I want to integrate the JavaScript web3modal in one of my client and for the I am using Angular 16 in as frontend app, but after installing the @web3modal/ethers5 ethers@5.7.2, my angular application starting throwing the error for web3modal internal modules 

`Error: node_modules/@web3modal/ethers5/dist/types/src/client.d.ts:20:11 - error TS2430: Interface 'Web3ModalState' incorrectly extends interface 'PublicStateControllerState'.
  Types of property 'selectedNetworkId' are incompatible.
    Type 'number | undefined' is not assignable to type '`${string}:${string}` | undefined'.
      Type 'number' is not assignable to type '`${string}:${string}`'.

20 interface Web3ModalState extends PublicStateControllerState {
             ~~~~~~~~~~~~~~

37     subscribeState(callback: (state: Web3ModalState) => void): () => void;

Attached screenshot for reference 
<img width="956" alt="Screenshot 2023-12-29 191834" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/75242338/8de742fe-11c0-4a6b-8213-15e1c7ac50e6">

`

**Comment by glitch-txs** [comment, marked as answer]
try adding `"skipLibCheck": true` in tsconfig.json

**Reply by rahulkm561** [reply]
Yes its resolved
